src/platform/intel/cavs/lib/pm_runtime.c

```diff
@@ -606,6 +609,17 @@ bool platform_pm_runtime_is_active(uint32_t context, uint32_t index)
 void platform_pm_runtime_power_off(void)
 {
 	uint32_t hpsram_mask[PLATFORM_HPSRAM_SEGMENTS], i;
+#if CAVS_VERSION >= CAVS_VERSION_1_8
+	int ret;
+
+	/* check if DSP is busy sending IPC for 10ms */
+	ret = poll_for_register_delay(IPC_HOST_BASE + IPC_DIPCIDR,
+				      IPC_DIPCIDR_MSG_MASK, ~IPC_DIPCIDR_BUSY,
+				      10000);
```

Comment:
Actually I am not very familiar with this function call, what timeout value would be better and if my usage is right here?
@lgirdwood @keyonjie @mmaka1 @kv2019i

Reply:
usage is right and 10ms seems fine.

Reply:
oops, I checked the latest test result with old test. It seems every CTX_SAVE all delayed by 10ms.
I will double check if this 10ms is the timeout in my code

old FW
```
[    6.985709] sof-audio-pci 0000:00:1f.3: ipc tx: 0x40010000: GLB_PM_MSG: CTX_SAVE
[    6.985965] sof-audio-pci 0000:00:1f.3: ipc tx succeeded: 0x40010000: GLB_PM_MSG: CTX_SAVE
```
new FW

```
[  154.212800] sof-audio-pci 0000:00:1f.3: ipc tx: 0x40010000: GLB_PM_MSG: CTX_SAVE
[  154.223116] sof-audio-pci 0000:00:1f.3: ipc tx succeeded: 0x40010000: GLB_PM_MSG: CTX_SAVE
```